app/api/tool_utilities.py

```diff
@@ -125,11 +126,12 @@ def finalize_inputs(input_data, validate_data: List[Dict[str, str]]) -> Dict[str
 
 def execute_tool(tool_id, request_inputs_dict):
     try:
+        tools_config = load_config()
```

Comment:
Rename this variable as it is very similar to the other variable: `tool_config`. This is good for having an intuitive understanding of the main flow of the code.